mParticle-iterable/MPKitIterable.m

```diff
@@ -25,25 +33,19 @@ + (void)load {
 
 + (void)setCustomConfig:(IterableConfig *_Nullable)config {
     _customConfig = config;
-    _customUrlDelegate = config.urlDelegate;
```

Comment:
Why did you remove this bug fix? Removing this introduces a crash regression. See the previously merged PR where this was implemented: https://github.com/mparticle-integrations/mparticle-apple-integration-iterable/pull/9